sentiment-analysis/src/test/java/StanfordSentimentTest.java

```diff
@@ -0,0 +1,38 @@
+import static org.junit.Assert.*;
+
+import org.alfresco.ml.sentimentanalysis.SentimentAnalyzer;
+import org.alfresco.ml.sentimentanalysis.stanford.StanfordAnalyzer;
+
+import junitparams.FileParameters;
+import junitparams.JUnitParamsRunner;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class StanfordSentimentTest {
+	SentimentAnalyzer a = new StanfordAnalyzer();
+	String[] outputs = new String[] {"Negative", "Neutral", "Positive"};
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	@FileParameters("src/test/resources/testdata.csv")
+	public void test(int sentiment, String data) {
```

Comment:
Lets create a simpler test with correct asserstion

Reply:
is this better?